tests/integration/test_github.py

```diff
@@ -182,7 +182,7 @@ def test_create_fork(colin_project):
 
 def test_is_fork(colin_project):
     assert not colin_project.is_fork
-    assert colin_project.is_forked()
+    assert colin_project.is_forked() is True
```

Comment:
seriously?

Reply:
^

Reply:
me not satisfakovan
```python
    def is_forked(self) -> bool:
        return bool(self._construct_fork_project())
```

Reply:
now I don't understand, why not check directly for True?

Reply:
no problem if you add a comment explaining it, otherwise I'd be WTF next time I see it

Reply:
okay, let's compromise: let's have a full, healthy goat and also be fed